modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc

```diff
@@ -1,34 +1,28 @@
 = File Firewall
 :toc: right
+:description: The File Firewall lets you control access and sharing in fine detail by creating rules for allowing or denying access.
 :regex-info-url: http://www.regular-expressions.info/
 :supported-mimetype-list-url: https://github.com/owncloud/core/blob/master/resources/config/mimetypemapping.dist.json
 
 == Introduction
 
-The File Firewall GUI enables you to manage firewall rule sets. 
-You can find it in your ownCloud admin page, under menu:Admin[Security]. 
-The File Firewall lets you control access and sharing in fine detail, by creating rules for allowing or denying access restrictions based on: _group_, _upload size_, _client devices_, _IP address_, _time of day_, as well as many more criteria. 
+{description} This is based on: _group_, _upload size_, _client devices_, _IP address_, _time of day_, as well as many more criteria. The File Firewall GUI enables you to manage the firewall rule sets. You can find it in your ownCloud admin page, under menu:Admin[Security]. 
+
 In addition to these restriction options, the File Firewall app also supports rules based on {regex-info-url}[regular expressions].
 
 == How the File Firewall Works
 
-Each firewall rule set consists of one or more conditions. 
-If a request matches all of the conditions, in at least one rule set, then the request is blocked by the firewall. 
-Otherwise, the request is allowed by the firewall.
+Each firewall rule set consists of one or more conditions. If a request matches all of the conditions, in at least one rule set, then the request is blocked by the firewall. Otherwise, the request is allowed by the firewall.
```

Comment:
```suggestion
Each firewall rule set consists of one or more conditions. If a request matches all of the conditions in at least one rule set, then the request is blocked by the firewall. Otherwise, the request is allowed by the firewall.
```